src/constants.js

```diff
@@ -28,14 +27,6 @@ export const STAGE = 'mainnet';
  */
 export const NETWORK = 'mainnet';
```

Comment:
Do we still need this constant? Should we rename it to `DEFAULT_NETWORK` or something like that?

If not, it is still being used in `./src/sagas/networkSettings.js`

Reply:
Yes, we need it. However, your suggestion is good. I changed to `NETWORK_MAINNET` to keep consistency with `NETWORK_TESTNET` constant.

See: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/376/commits/30152fcab3314d8208acdfd3fd503747be4094dd